### ISSUE TITLE
How can I handle multiple input component ?

### ISSUE THREAD

**Issue by ajallon-sqli** [NewOldMax/react-form-validator-core#52]
Hi guys, first thank you for your work. Your package is the only one that covered all my uses cases... Until now.
 I'm trying to make a Radio ValidatorComponent (RadioGroupValidator) that contain some Radio components (`<input type="radio"  {...props}  />`). I use the "required" rule with an error message as it is a radio (I just want to check if at least on radio has been checked).
The problem is that when i submit my form, the error appears event if one of my radio is checked.
I make the error disappear with the "makeValid" methode of ValidatorComponent but when validation is call it still returns an error.

### TL;DR:
- Is it possible to handle multiple inputs component with react-form-validator-core?
(radio, checkbox...)

- If yes can you give an example ?

**Comment by NewOldMax** [comment]
Hi,
can you reproduce your issue at https://codesandbox.io/s/new ?

**Comment by ajallon-sqli** [comment]
Here is it:
https://codesandbox.io/s/hopeful-sea-x8t64

**Comment by NewOldMax** [comment]
Try this one https://codesandbox.io/s/f64c7

**Comment by ajallon-sqli** [comment]
Thank you for the fast answer, didn't thought about that solution it's awesome ^^
